modules/programs/gitui/gitui.nix

```
@@ -0,0 +1,52 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.gitui;
+
+in {
+  meta.maintainers = [ hm.maintainers.mifom ];
+
+  options.programs.gitui = {
+    enable =
+      mkEnableOption "gitui, blazing fast terminal-ui for git written in rust";
+
+    keyConfig = mkOption {
+      type = types.either types.path types.lines;
+      default = ./default_key_config.ron;
```

```suggestion
      default = ./default_key_config.ron;
      defaultText = literalExpression (concatStringsSep "\n" [ "''" (builtins.readFile ./default_key_config.ron) "''"]);
```

I think this should render the default configuration correctly.